modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -0,0 +1,229 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+:_content-type: PROCEDURE
+[id="virt-measuring-latency-vm-secondary-network_{context}"]
+= Verifying connectivity and measuring latency between virtual machines on a secondary network
```

Comment:
Eventually there will be quite a few cluster checkups, so I think it would be a good idea to shorten the module titles. Otherwise the TOC will become difficult to read.
 
Suggestion: "Checking network connectivity and latency for virtual machines on a secondary network"